hacks/update_role_init_part/commons.py

```
@@ -10,36 +10,29 @@ def find_root_directory(cur_dir):
-
-with open(f"config.txt", "r") as file:
-    config_dict_str = file.read()
-    config_dict = eval(config_dict_str)
-
 # Load python utils
 script_dir = os.path.join(root_directory, "commons", "python")
-sys.path.append(root_directory)
+src_dir = os.path.join(root_directory, "src")
+sys.path.append(src_dir)
 sys.path.append(script_dir)
 import py_utils
```

Ruff (0.8.2)

`py_utils` imported but unused

Remove unused import: `py_utils`

(F401)